test/jdk/tools/launcher/ArgsEnvVar.java

```diff
@@ -218,6 +218,24 @@ public void openQuoteShouldFail() {
         }
     }
 
+    @Test
+    public void NoOptionsDuplication() {
+        if (System.getProperty("os.family") == "windows") {
+           /* This test has no value on windows */
+           return;
+        }
+        env.put(JDK_JAVA_OPTIONS, "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005");
```

Comment:
Tests uses hard coded ports are problematic as they will fail if the port is in used by something else, like another tests running concurrently.